src/components/BooksList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+
+// end imports
+
+const BookList = () => {
+  const books = useSelector((state) => state.books.books);
+  const dispatch = useDispatch();
+
+  const handleRemoveBook = (id) => {
+    dispatch(removeBook({ index: id }));
+  };
+
+  return (
+    <div>
+      <h2>Book List</h2>
+      <ul>
+        {books.map((book) => (
+          <li key={book.item_id}>
+            {book.title}
+            {' '}
+            by
+            {book.author}
+            <button type="button" onClick={() => handleRemoveBook(book.item_id)}>Remove Book</button>
```

Comment:
- [ ] Great job implementing the remove button, however, please be informed that the remove book button does not remove the books added i.e it is not working


Remember, you're making good progress! Feel free to refer to the provided resources for additional guidance on implementing the necessary methods and relationships. Keep up the good work!